src/components/pages/AccountPage/AccountPage.js

```diff
@@ -45,7 +45,7 @@ const AccountPage = props => {
     axiosWithAuth()
       .put(`/profile/${updatedUserInfo.user_id}`, updatedUserInfo)
       .then(res => {
-        setHrfUserInfo(res.data.updated_profile);
+        setHrfUserInfo(updatedUserInfo);
       })
       .catch(err => console.log(err));

```

Comment:
Do we want to log the err for the client? Could be helpful to keep for now while we're developing.

Reply:
Good point. We may need to go through at some point to remove console logs like these. But like you said, they are helpful during development.